buttons/src/main/res/values/attrs.xml

```diff
@@ -1,8 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <attr name="buttonReverseStyle" format="reference" />
```

Comment:
Same here, to avoid breaking change, we need to keep the declaration but redirect the implementation to the new references inside the Vitamin theme style.

Reply:
<!-- @deprecated -->

Reply:
Fixed